### PR TITLE
Purge newsletter support

### PR DESCRIPTION
**Pull request by mkoistinen**
As agreed, removing newsletter support.

**IMPORTANT NOTE: *Before* this gets merged, please tag master with the latest release (incrementing if necessary). This will be useful to anyone using newsletters and wanting the latest, pre-purge code.**

Here's an updated model-relationship diagram.
20150714b aldryn jobs db schema

### CODE REVIEW

**Comment by intellectronica** [comment]
:+1:

File: aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/plugins/latest_entries.html [hunk 1]
```diff
@@ -7,17 +7,18 @@
         {% endif %}
     {% else %}
         <p class="alert alert-info">
-            {% blocktrans with total=instance.job_openings.count %}
-                Showing {{ total }} Job Openings
-            {% endblocktrans %}
++            {% blocktrans count counter=vacancies_count %}
```
Comment (czpython) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/plugins/latest_entries.html]: need to remove ```+``` here :)

Comment (mkoistinen) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/plugins/latest_entries.html]: Oops... on it.

File: aldryn_jobs/cms_plugins.py [hunk 2]
```diff
@@ -57,32 +55,18 @@ class JobList(NameSpaceCheckMixin, CMSPluginBase):
...
+        namespace = (instance.app_config.namespace if instance.app_config
+                     else '')
+        if namespace == '' or context.get('plugin_configuration_error', False):
+            vacancies = JobOpening.objects.none()
         else:
-            context['form'] = NewsletterSignupForm(
-                app_config=instance.app_config)
+            vacancies = instance.job_openings(namespace)
+        context['vacancies'] = vacancies
+        context['vacancies_count'] = vacancies.count()
```
Comment (czpython) [inline aldryn_jobs/cms_plugins.py]: best to use ```len(vacancies)``` because otherwise it would execute two queries instead of one.

**Comment by mkoistinen** [comment]
Had to rebase a little anyway =/

**Comment by FinalAngel** [comment]
Awesome :+1:

**Pull request merged**